static/js/message_edit.js

```diff
@@ -421,7 +421,16 @@ export function get_available_streams_for_moving_messages(current_stream_id) {
         .map((stream) => ({
             name: stream.name,
             value: stream.stream_id.toString(),
-        }));
+        }))
+        .sort((a, b) => {
+            if (a.name.toString().toLowerCase() < b.name.toString().toLowerCase()) {
+                return -1;
+            }
+            if (a.name.toString().toLowerCase() > b.name.toString().toLowerCase()) {
+                return 1;
+            }
+            return 0;
```

Comment:
Why is `toString` required? I'd expect the names to already be strings.

Reply:
I don't know why, but when I added the .toLowerCase() function it didn't work, an error occurred saying a.name don't have this function.

Reply:
I don't know why but at some point of the development an error occurred saying a.name wasn't a string at all. I made an test here and it looks like this error is no longer a problem. I'll fix this, thanks for the review.